packages/puppeteer-core/src/bidi/core/Session.ts

```diff
@@ -29,20 +28,21 @@ export class Session
     capabilities: Bidi.Session.CapabilitiesRequest
   ): Promise<Session> {
     // Wait until the session is ready.
-    let status = {message: '', ready: false};
-    for (let i = 0; i < MAX_RETRIES; ++i) {
-      status = (await connection.send('session.status', {})).result;
-      if (status.ready) {
-        break;
-      }
-      // Backoff a little bit each time.
-      await new Promise(resolve => {
-        return setTimeout(resolve, (1 << i) * 100);
-      });
-    }
-    if (!status.ready) {
-      throw new Error(status.message);
-    }
+    // TODO: Once Chrome supports session.status properly, uncomment this block.
+    // let status = {message: '', ready: false};
+    // for (let i = 0; i < MAX_RETRIES; ++i) {
```

Comment:
let's only run this if the browser is not Chrome instead of leaving it commented out?

Reply:
Looks like this logic is placed in `BrowserConnector` already. We would need to remove the logic there and replace it with this one once we support pure bidi.

Reply:
I've updated the comment with this information.